Formula/lastpass-cli.rb

```diff
@@ -18,13 +18,27 @@ class LastpassCli < Formula
   depends_on "cmake" => :build
   depends_on "docbook-xsl" => :build
   depends_on "pkg-config" => :build
+  depends_on "curl" if MacOS.version >= :mojave
   depends_on "openssl"
   depends_on "pinentry" => :optional
 
   def install
     ENV["XML_CATALOG_FILES"] = etc/"xml/catalog"
-    system "make", "PREFIX=#{prefix}", "install"
-    system "make", "MANDIR=#{man}", "install-doc"
+
+    args = std_cmake_args + %W[
+      -DCMAKE_INSTALL_MANDIR:PATH=#{man}
+    ]
+
+    if MacOS.version >= :mojave
+      curl_prefix = Formula["curl"].opt_prefix
+      args << "-DCMAKE_PREFIX_PATH=#{curl_prefix}"
```

Comment:
`CMAKE_PREFIX_PATH` is the wrong variable to set (it affects all search paths).
I think `-DCURL_ROOT=#{Formula["curl"].opt_prefix}` should work nicely.

Reply:
Also, please add a comment above the `curl` dependency, explaining it is a temporary solution and giving the link to the upstream issue.

Reply:
Is this even necessary? I was under the impression that homebrew sets `CMAKE_PREFIX_PATH` to `brew --prefix` and lumps in keg-only dependencies into `CMAKE_PREFIX_PATH` as well. Have you tested this without including this `if` block?